x-pack/plugins/fleet/common/openapi/paths/health_check.yaml

```diff
@@ -10,12 +10,13 @@ post:
           schema:
             type: object
             properties:
-              name:
-                type: string
               status:
                 type: string
+              id:
+                type: string
```

Comment:
nit: we could add a description on what the `id` represents.

Strictly speaking introducing a new required parameter is a breaking API change, should we make it optional? It could happen that customers/other projects are using it.